Documentation/En/Index.xyl

```diff
@@ -14,7 +14,7 @@
 
   <h2 id="Introduction" for="main-toc">Introduction</h2>
 
-  <p>Software quality is a vast topic in Computer Science. There is a lot of
+  <p>Software quality is a vast topic in Computer Science. There are a lot of
```

Comment:
Hmm, yeah if `lot` is the subject, then `is` is OK. But as an English reader this sounds strange and I have to read it and emphasise "lot". For me it would be less ambiguous as:

````
There are many strategies
````